config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -497,6 +497,10 @@ periodics:
   interval: 48h
   agent: kubernetes
   decorate: true
+  extra_refs:            # Periodic job doesn't clone any repo by default, needs to be added explicitly
+    - org: cert-manager
+      repo: test-infra
+      base_ref: master
```

Comment:
Could we also clone the cert-manager repo in the same way? (Both here and in the presubmit job.)